tests/foreman/ui/test_contentview_new.py

```diff
@@ -0,0 +1,44 @@
+"""Test class for Content View UI
```

Comment:
As new CV view is default, so I think we should rename this as test_contentview.py, instead renaming old module with suffix as _old, wdyt?